core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -176,7 +177,11 @@ class ReplicaFetcherThread(name: String,
     // traffic doesn't exceed quota.
     if (quota.isThrottled(topicPartition))
       quota.record(records.sizeInBytes)
-    replicaMgr.brokerTopicStats.updateReplicationBytesIn(records.sizeInBytes)
+
+    if (partition.isReassigning && partition.isAddingLocalReplica)
+      replicaMgr.brokerTopicStats.updateReassignmentBytesIn(records.sizeInBytes)
+    else
+      replicaMgr.brokerTopicStats.updateReplicationBytesIn(records.sizeInBytes)
```

Comment:
Similar comment. We should also count reassignment traffic toward replication traffic.